library/classes/html2text.class.php

```diff
@@ -34,30 +34,30 @@
 /**
  *  Takes HTML and converts it to formatted, plain text.
  *
- *  Thanks to Alexander Krug (http://www.krugar.de/) to pointing out and
+ *  Thanks to Alexander Krug (https://www.krugar.de/) to pointing out and
```

Comment:
This link only works with http , so needs to stay as http

Reply:
👍